src/core/Akka.Persistence/Eventsourced.cs

```diff
@@ -81,6 +81,7 @@ public abstract partial class Eventsourced : ActorBase, IPersistentIdentity, IPe
         private readonly IStash _internalStash;
         private IActorRef _snapshotStore;
         private IActorRef _journal;
+        private IActorRef _recoveryPermitter;
```

Comment:
The recovery permitter is being cached here instead of in the `Persistence` extension. This is lazily created when the plugin is initialized for the first time.